great_expectations/rule_based_profiler/domain_builder/domain_builder.py

```diff
@@ -14,7 +17,7 @@
 from great_expectations.validator.validator import Validator
 
 
-class DomainBuilder(ABC):
+class DomainBuilder(SerializableBuilder, ABC):
```

Comment:
Why is this serializable?

Reply:
@donaldheppner Because it implements `to_json_dict()`.